Index: git/src/agent/otbr-agent.default.in
===================================================================
--- git.orig/src/agent/otbr-agent.default.in
+++ git/src/agent/otbr-agent.default.in
@@ -1,5 +1,5 @@
 # Default settings for otbr-agent. This file is sourced by systemd
 
 # Options to pass to otbr-agent
-OTBR_AGENT_OPTS="-I wpan0 -B @OTBR_INFRA_IF_NAME@ spinel+hdlc+uart:///dev/ttyACM0 trel://@OTBR_INFRA_IF_NAME@"
+OTBR_AGENT_OPTS="-I wpan0 -B @OTBR_INFRA_IF_NAME@ spinel+hdlc+uart:///dev/ttyUSB0 trel://@OTBR_INFRA_IF_NAME@"
 OTBR_NO_AUTO_ATTACH=@OTBR_NO_AUTO_ATTACH@
